src/spokes/PoolManager.sol

```diff
@@ -66,8 +64,6 @@ contract PoolManager is
     ITokenFactory public tokenFactory;
     IVaultMessageSender public sender;
     IPoolEscrowFactory public poolEscrowFactory;
-    IAsyncRequestManager public asyncRequestManager;
```

Comment:
Only non-renaming in this PR is the removal of these 2 variables.

Reply:
Great catch, thanks for the cleanup!